src/unity/toolkits/object_detection/one_shot_object_detection/util/mapping_function.hpp

```diff
@@ -44,7 +44,7 @@ boost::gil::point2<F> transform(const Eigen::Matrix<F, 3, 3>& mat, const boost::
 
 template <typename F>
 struct mapping_traits<Eigen::Matrix<F, 3, 3> >{
-    typedef boost::gil::point2<F> result_type;
+  typedef boost::gil::point2<F> result_type;
```

Comment:
In new code, let's use using declarations instead of typedef.

See also: https://google.github.io/styleguide/cppguide.html#Aliases

Reply:
Fixed.